monarch_ingest/omim/gene-disease.py

```diff
@@ -0,0 +1,161 @@
+gene_id = None
+
+if disorder_match is not None:
+    disorder_parts = disorder_match.groups()
+    disorder_label, disorder_num, association_key = disorder_parts
+
+    gene_symbols = gene_symbols.split(', ')
+    gene_id = 'OMIM:' + str(gene_num)
+    disorder_id = 'OMIM:' + disorder_num
+
+    gene = Gene(id=gene_id)
+    koza_app.write(gene)
+
+elif nogene_match is not None:
+    # this is a case where the disorder
+    # a blended gene/phenotype
+    # we lookup the NCBIGene feature and make the association
+    disorder_label, association_key = nogene_match.groups()
+    # make what's in the gene column the disease
+    disorder_id = 'OMIM:' + gene_num
+    ncbi_id = ''
+    if gene_num in omim_to_gene:
+        ncbi_id = omim_to_gene[gene_num]['Entrez Gene ID (NCBI)']
+    if ncbi_id == '':
+        koza_app.next_row()
+    gene_id = 'NCBIGene:' + ncbi_id
+
+    genomic_entity = NucleicAcidEntity(
+        id=gene_id,
+        type=koza_app.translation_table.global_table['heritable_phenotypic_marker'],
+    )
+
+else:
+    # In dipper we created anonymous nodes for these cases, but for now we will skip these
+    # It doesn't appear that anything falls into this category as of 1/11/2021
+    LOG.warning(f"skipped {row}")
+    koza_app.next_row()
+
+
+# From the docs:
+
+# Brackets, "[ ]", indicate "nondiseases," mainly genetic variations
+# that lead to apparently abnormal laboratory test values
+
+# (e.g., dysalbuminemic euthyroidal hyperthyroxinemia).
+# Braces, "{ }", indicate mutations that contribute to susceptibility
+# to multifactorial disorders (e.g., diabetes, asthma) or to
+# susceptibility to infection (e.g., malaria).
+#
+# A question mark, "?", before the phenotype name indicates that the
+# relationship between the phenotype and gene is provisional.
+# More details about this relationship are provided in the comment
+# field of the map and in the gene and phenotype OMIM entries.
+
+# Phene key:
+# The number in parentheses after the name of each disorder indicates
+# the following:
+#   (1) the disorder was positioned by mapping of the wildtype gene;
+#   (2) the disease phenotype itself was mapped;
+#   (3) the molecular basis of the disorder is known;
+#   (4) the disorder is a chromosome deletion or duplication syndrome.
+# reference: https://omim.org/help/faq#1_6
+
+predicate = Predicate.causes
+relation = koza_app.translation_table.global_table['causes condition']
+
+if disorder_label.startswith('['):
+    predicate = Predicate.related_to
+    relation = koza_app.translation_table.global_table['is marker for']
+elif disorder_label.startswith('{'):
+    predicate = Predicate.contributes_to
+    relation = koza_app.translation_table.global_table['contributes to']
+elif disorder_label.startswith('?'):
+    # this is a questionable mapping!  skip?, skipping for now
+    # predicate = Predicate.contributed_to
+    # relation = koza_app.translation_table.global_table['contributes to']
+    koza_app.next_row()
```

Comment:
@sabrinatoro curious your thoughts on this section, wondering if I should instead be using these biolink predicates:
 
- https://biolink.github.io/biolink-model/docs/gene_associated_with_condition.html
- https://biolink.github.io/biolink-model/docs/risk_affected_by.html
- https://biolink.github.io/biolink-model/docs/related_condition.html

I'm guessing these should all be gene_associated_with_condition and we should use qualifiers for these different levels?